source/layouts/layout.erb

```diff
@@ -29,6 +29,13 @@
     <a href="#main" class="skip-link">Skip to main content</a>
 
     <header class="header<% if current_page.url == '/' %> header--without-border<% end %>">
+
+      <div class="govuk-phase-banner">
```

Comment:
Is _Phase_ the correct banner type for cookies? I thought it should be used to display the service phase (eg. beta).

The design system cookie banner doesn't use the same styling, should we be consistent with other services?
```html
<div class="app-cookie-banner js-cookie-banner" style="display: block;">
  <p class="govuk-width-container">GOV.UK uses cookies to make the site simpler. <a href="/cookies" class="govuk-link">Find out more about cookies</a></p>
</div>
```

Reply:
Thanks!